openreview/conference/builder.py

```diff
@@ -340,6 +343,10 @@ def set_submission_revision_stage(self, stage):
 
     def set_decision_stage(self, stage):
         self.decision_stage = stage
+
+        if self.decision_stage.decisions_file is not None:
```

Comment:
This might be minor because I am not sure in what scenario this would happen but:
if the PCs upload a decisions file and for some reason it does not work and then try to just run the decision stage without a file (clicking the X next to upload_decisions), then `self.decision_stage_file` will be `''` instead of `None`, and there will be an error when we try to get the file.